src/main/cpp/include/api/genomicsdb.h

```diff
@@ -281,6 +282,7 @@ class GENOMICSDB_EXPORT GenomicsDBVariantProcessor {
   void initialize(std::map<std::string, genomic_field_type_t> genomic_field_types) {
      m_genomic_field_types = std::make_shared<std::map<std::string, genomic_field_type_t>>(std::move(genomic_field_types));
   }
+  void initialize(const VariantQueryConfig &query_config, void *annotation_service);
```

Comment:
I am a little ambivalent about overloading initialize(). Maybe, we should keep this incarnation and do away with the other one as and when we port the processors in the `cython`, `R` and `Go` bindings. Can you make a TODO note of that?